src/background/deinflect.ts

```diff
@@ -510,19 +453,28 @@ export function deinflect(word: string): CandidateWord[] {
     // final る. Since the stem is shorter than the plain form, to avoid
     // adding multiple entries for all possible stem variations to the rule
     // data array, we forward the stem to the plain form programmatically.
-    if (type & (Type.MasuStem | Type.TaTeStem)) {
+    if (type & (Type.MasuStem | Type.TaTeStem | Type.IrrealisStem)) {
       const reason = [];
 
       // Add the "masu" reason only if the word is solely the masu stem.
       if (type & Type.MasuStem && !thisCandidate.reasonChains.length) {
         reason.push([Reason.MasuStem]);
       }
 
-      result.push({
-        word: word + 'る',
-        type: Type.IchidanVerb | Type.KuruVerb,
-        reasonChains: [...thisCandidate.reasonChains, ...reason],
-      });
+      // Ichidan verbs inflect to られる and させる, which exist as sperate
+      // rules in the rule data array. れる and せる are not applicable.
```

Comment:
I don't think I understood this comment. Perhaps it seems some more context?

Reply:
Sure, I can see that it is a little bit too scarce.
```ts
      // Ichidan verbs attach the auxiliary verbs られる and させる instead of
      // れる and せる for the passive and causative forms to their stem. Since
      // られる and させる exist as separate rules that bypass the irrealis stem
      // type, we ignore the the rules with a to-type of IrrealisStem for the
      // passive and causative, i.e. the rules for れる and せる.
```
Does this help clarify the context?

Reply:
Perfect! Thank you!